src/core_inner/backend/error.rs

```diff
@@ -83,15 +83,50 @@ impl Into<Status> for Error {
         //TODO add custom errors
         trace!("Error: {}", self.clone());
         match self {
-            Self::KeyNotFound(key) => Status::not_found(&format!("KeyNotFound: {}", key)),
+            Self::KeyNotFound(key) => Status::not_found(format!("KeyNotFound {}", key)),
             Self::DuplicateKey => Status::already_exists("DuplicateKey"),
-            _ => Status::unknown("Other errors"),
+            Self::Timeout => Status::deadline_exceeded("Timeout"),
+            Self::VDiskNoFound(id) => Status::not_found(format!("VDiskNoFound {}", id)),
+            Self::Storage(msg) => Status::internal(format!("Storage {}", msg)),
+            Self::VDiskIsNotReady => Status::internal("VDiskIsNotReady"),
+            Self::Failed(msg) => Status::internal(format!("Failed {}", msg)),
+            Self::Internal => Status::internal("Internal"),
+            Self::PearlChangeState(msg) => Status::internal(format!("PearlChangeState {}", msg)),
         }
     }
 }
 
 impl From<Status> for Error {
     fn from(status: Status) -> Self {
-        Self::Failed(format!("{:?}", status))
+        let mut words = status.message().split_whitespace();
```

Comment:
Why do we need this?

Reply:
Format of messages: Type <optional args>. Can be easily separated by whitespaces